ruma-identity-service-api/src/lookup/lookup_3pid/v2.rs

```diff
@@ -0,0 +1,52 @@
+//! [POST /_matrix/identity/v2/lookup](https://matrix.org/docs/spec/identity_service/latest#post-matrix-identity-v2-lookup)
```

Comment:
Could you change this to use the current version rather than `latest`? (I know I put `latest` in the issue, so that's my bad.)

```suggestion
//! [POST /_matrix/identity/v2/lookup](https://matrix.org/docs/spec/identity_service/0.3.0#post-matrix-identity-v2-lookup)
```